docker/collect_static.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 source /opt/openoni/ENV/bin/activate
-django-admin.py collectstatic --noinput
+/opt/openoni/manage.py collectstatic --noinput
```

Comment:
It's in one of the commits' notes - django-admin.py only worked previously because of a horrible hack that forced Python to see /opt as the root of anything referencing `openoni.*`.  That hack made scripts slightly easier, but the new django `manage.py` magicks that away for us anyway, *and* that hack only works if we hack it for every single namespace we want.  So in killing that hack, I also had to make use of `manage.py`.

The good news is that `manage.py` is the way these kinds of tasks are supposed to run anyway.